www/dfe_shiny_gov_style.css

```
@@ -409,4 +409,52 @@ html {
 .small-box.bg-dark-blue {
   background-color: #12436D !important;
   color: #ffffff !important;
+}
+
+/* Set default reactable styles */
+.reactable {
+  font-size: 1rem !important;
+  table-layout: fixed;
+  width: 100% !important;
+  word-break: keep-all; /* Prevents breaking words mid-word */
+  white-space: normal; /* Allows line breaks between words */
+}
+
```

Initial css to back up the change requested in the reactable in server.R

```suggestion
.bar-sort-header:hover,
.bar-sort-header:focus {
  background: #fd0;
}

/* Add a top bar on ascending sort */
.bar-sort-header[aria-sort="ascending"] {
  box-shadow: inset 0 0.188rem 0 0 #1d70b8;
}

/* Add a bottom bar on descending sort */
.bar-sort-header[aria-sort="descending"] {
  box-shadow: inset 0 -0.188rem 0 0 #1d70b8;
}

/* Add an animation when toggling between ascending and descending sort */
.bar-sort-header {
  transition: box-shadow 0.3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
}

/* Table row highlighting -------------------------------------------------- */
.rt-tr-highlight:hover {
    background-color: #fd0;
}
```